HereBeDragons-1.0.lua

```diff
@@ -308,6 +308,53 @@ if not oldversion or oldversion < 20 then
                 mapData[remapID] = mapData[validMapID]
             end
         end
+		
+        local continentSizes = {
+            [WORLDMAP_AZEROTH_ID] = {
+                [1] = 63570, -- Eastern Kingdoms, or floor 0
+                [2] = 42382,
+                [3] = 53730,
+                [4] = 19600,
+                floors = {
+                    [0] = { 1,1,1,1, "just here to fool some functions that check floors[num]" },
+                    [1] = { -- Kalimdor
+                        [1] = 65700,
+                        [2] = 43795,
+                        [3] = 11900,
+                        [4] = 23760,
+                    },
+                    [571] = { -- Northrend
+                        [1] = 65700,
+                        [2] = 43795,
+                        [3] = 33440,
+                        [4] = 11960,
+                    },
+                    [870] = { -- Pandaria
+                        [1] = 58520,
+                        [2] = 39015,
+                        [3] = 29070,
+                        [4] = 34410,
+                    },
+                    [1220] = { -- Broken Isles
+                        [1] = 96710,
+                        [2] = 64476,
+                        [3] = 63100,
+                        [4] = 29960
+                    }
```

Comment:
If these floor data tables would set the "instance" member to the appropriate value, then some extra functions like TranslateZoneCoordinates and GetZoneDistance could actually work with the world map, at least in theory, I didn't test it yet.